_components/site-alert/guidance/accessibility.md

```diff
@@ -1 +1,37 @@
-- **Don’t visually hide alert messages and then make them visible when they are needed.** Users of older assistive technologies may still be able to perceive hidden alert messages. Fully remove alert messages when they’re not needed.
\ No newline at end of file
+- **Don’t visually hide alert messages and then make them visible when they are needed.** Users of older assistive technologies may still be able to perceive hidden alert messages. Fully remove alert messages when they’re not needed.
+- **Include a descriptive ARIA label.** Add an `aria-label` or `aria-labelledby` attribute to the `.usa-site-alert` element to ensure that your site alert appears in the landmarks menu. A well-written label can help assistive technology users easily find and understand the top-level information of your site alert.
+- **Add an ARIA role to dynamically-updated site alerts.** If your site alert will be dynamically updated during a session, add an ARIA `role` attribute to notify assistive technologies users of important and time-sensitive changes. To elevate importance, add the appropriate `role` to the `.usa-site-alert` element, chosen from the following table:
```

Comment:
Suggesting: "... will be dynamically updated during a session, for example when a user does XYZ,"

to provide additional context for how this could be used.

Reply:
I reorganized the aria role section a bit for clarity to more accurately explain the use case. I also added an example as suggested.  [39c6fe0](https://github.com/uswds/uswds-site/pull/1774/commits/39c6fe08b82b1d2978ef420698f257de554a7484)